docs/metrics.md

```diff
@@ -139,9 +139,10 @@ Indication for a VirtualMachine that its eviction strategy is set to Live Migrat
 Indication for the total number of VirtualMachineInstance workloads that are not running within the most up-to-date version of the virt-launcher environment. Type: Gauge.
 
 ### kubevirt_vmi_phase_count
-Sum of VMIs per phase and node.
+Sum of VMIs per phase and node. `phase` can be one of the following: [`Pending`, `Scheduling`, `Scheduled`, `Running`, `Succeeded`, `Failed`, `Unknown`]. Type: Gauge.
 
-`phase` can be one of the following: [`Pending`, `Scheduling`, `Scheduled`, `Running`, `Succeeded`, `Failed`, `Unknown`]. Type: Gauge.
+### kubevirt_vmi_running_creation_timestamp_second
+VirtualMachineInstance creation timestamp. Type: Counter.
```

Comment:
I'm not understanding what this metric is.  Would this share every vmi's creation timestamp with prometheus?

Can you also explain how you'd use this metric?

Reply:
This might be relevant: https://github.com/kubevirt/kubevirt/pull/7918#issuecomment-1196409004.

Reply:
Yes, this metric will share every vmi's creation timestamp with prometheus. This metric can be used to check how old a vmi are.

Reply:
I am not sure if this metric make sense, but I think at least the metric name should be improved. I would not have guessed that. It is hard to understand what "timestamp seconds" are imo :)

Reply:
Can the metric be `kubevirt_vmi_creation_timestamp`?

Unfortunately the vmi phase transitions don't cover this use-case.

Reply:
> Can the metric be `kubevirt_vmi_creation_timestamp`?

_seconds suffix is a must.

> I tried to use the same pattern used here

Hm, ok then ...

Reply:
I thought `timestamp` was the unit?  `timestamp_seconds` seems redundant to me.

Reply:
oh, ok, why is it not done in seconds since it happened, like the others? That does not seem to be a prometheus way of doing things ...

Reply:
Right and I understand why: you need to make clear your unit of measurement. But we're not saying our timestamps are tracked in seconds, right?  Just seems confusing.

Reply:
I may have gotten a little lost, so apologies if the answer will be out of context.
Following the naming convention https://prometheus.io/docs/practices/naming/
there is a specific example with the timestamp.
I think that in this case we are doing things exactly how prometheus says to do things. Or am I missing something? Thanks

Reply:
Ya understood @fossedihelm.  All I'm saying is `timestamp_seconds` sounds confusing to me, even if that's convention.  I understand what you're doing with `kubevirt_vmi_creation_timestamp_seconds` and it lgtm.

Reply:
Thanks @rthallisey! And sorry if it sounds bad. As I said probably I gotten a little lost.
FWIW I agree with you, especially because they also says `...should use base units (e.g. seconds, bytes, meters - not milliseconds, megabytes, kilometers).` and base unit for Time is `seconds` https://prometheus.io/docs/practices/naming/#base-units